[2024-03-05 10:42:19,619] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project.merge_data 2024-03-05T10:42:13.512806+00:00 [queued]>
[2024-03-05 10:42:19,635] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project.merge_data 2024-03-05T10:42:13.512806+00:00 [queued]>
[2024-03-05 10:42:19,635] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-05 10:42:19,635] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-05 10:42:19,635] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-05 10:42:19,647] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-05T10:42:13.512806+00:00
[2024-03-05 10:42:19,652] {standard_task_runner.py:52} INFO - Started process 14987 to run task
[2024-03-05 10:42:19,655] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting_project', 'merge_data', '2024-03-05T10:42:13.512806+00:00', '--job-id', '294', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp17um3leu', '--error-file', '/tmp/tmphs3niy3x']
[2024-03-05 10:42:19,657] {standard_task_runner.py:77} INFO - Job 294: Subtask merge_data
[2024-03-05 10:42:19,697] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting_project.merge_data 2024-03-05T10:42:13.512806+00:00 [running]> on host 4622d7b8f726
[2024-03-05 10:42:19,751] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting_project
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-05T10:42:13.512806+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-05T10:42:13.512806+00:00
[2024-03-05 10:42:19,791] {python.py:151} INFO - Done. Returned value was: None
[2024-03-05 10:42:19,805] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting_project, task_id=merge_data, execution_date=20240305T104213, start_date=20240305T104219, end_date=20240305T104219
[2024-03-05 10:42:19,868] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-05 10:42:19,894] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
